dashboard/test/controllers/api/v1/sections_controller_test.rb

```diff
@@ -391,6 +391,16 @@ class Api::V1::SectionsControllerTest < ActionController::TestCase
     assert_equal false, returned_section.lesson_extras
   end
 
+  test 'default code_review_enabled value is FALSE' do
```

Comment:
Maybe I missed this (I think you mentioned during standup), but confused why we set code review enabled to true for every section in migration, yet want to set to false going forward? Feel like this could create weirdness down the line...